projects/igniteui-angular/src/lib/core/styles/components/navdrawer/_navdrawer-theme.scss

```diff
@@ -239,10 +239,10 @@
                 max-width: rem(32px);
                 // important is needed to override the typography margins
                 margin: rem(14px) auto !important;
+            }
 
-                igx-icon {
-                    margin-inline-start: 0;
-                }
+            igx-icon {
+                margin-inline-start: 0;
```

Comment:
Do we even need this now that we have the gap property set on the item?

Reply:
Yes, since the igx-icon does have margin-inline-start for indigo theme. I can move the reset specifically for indigo theme as it was.